samples/dashboards/demo/vertical/demo.html

```diff
@@ -0,0 +1,117 @@
+<script src="https://code.highcharts.com/highcharts.js"></script>
+<script src="https://code.highcharts.com/dashboards/datagrid.js"></script>
+<script src="https://code.highcharts.com/dashboards/dashboards.js"></script>
+<script src="https://code.highcharts.com/dashboards/modules/layout.js"></script>
+<script src="https://code.highcharts.com/modules/accessibility.js"></script>
+
+<div id="container">
+    <h1 id="title">Climate Change Over the Last 10 Decades: A Deep Dive into the Data</h1>
```

Comment:
Insert the attached image above the title (just to increase the "blogginess"). Below image insert a right aligned caption ( 0.9em #4d5156) that says "Amazing climate illustration, courtesy of DALL-E"

Title: change font-size to 3em and color to #4d5156 (a little easier on the eyes than the black one).

![climate](https://github.com/user-attachments/assets/b4956a07-f7af-4583-8bf0-6da5735e6af3)